### ISSUE TITLE
Transifex translations are not being added during a release

### ISSUE THREAD

**Issue by ghost** [silverstripe/silverstripe-mfa#441]
There is a German translation for this module in transifex, but the translations are not being pulled when the module gets new minor releases:

https://www.transifex.com/silverstripe/silverstripe-mfa/language/de/

**Comment by tom890180** [comment]
Hey,

We've translated this module to swedish and finnish as well, and we'd appreciate if these translations were pulled in. I would make a pull request myself but on `tx pull` I don't get any new ones (not even the german ones), so I assume they need to be reviewed first (?)

Tom

**Comment by NightJar** [comment]
@GuySartorelli Thanks for [updating the translations](https://github.com/silverstripe/silverstripe-mfa/issues/469). The OP here from @bitwingsde mentions that this has been an issue over several releases in the past though. Is there an issue with the release process that needs to be fixed?

**Comment by GuySartorelli** [comment]
@emteknetnz was the person who updated the translations and has done releases before whereas I have only supported.
To the best of my knowledge the release process will pull in translations, I can't speak for why this might not have happened in previous releases.

**Comment by emteknetnz** [comment]
I'm not 100% sure, though my best guess was there wasn't a minor release for the MFA module during the previous minor release (4.11), thus the translations were not run. At this stage translations are only regularly run and released during "official" minor releases where the sub modules (such as the MFA module) also have new minor releases created.

**Comment by NightJar** [comment]
Yeah, the translation update is run as part of the "main" release tool, `cow`. Minor releases tagged or created via GitHub won't have that step taken automatically and must be performed manually beforehand.

I thought `cow` checked for translations during the release process so would create a releasable commit "on the fly" during the process... but I'm probably remembering wrong & the tx update happens after the "should release" calculation.

Anyway... is there some kind of "before release" GitHub Action hook that could do a thing like that, so it isn't forgotten in the future? And/or pre-tag `git` hook?

Just brainstorming ideas at this point to try and avoid this in the future :)

**Comment by GuySartorelli** [comment]
Closing as we changed how we do this since this issue was opened and I can see there are several translation update commits since then.
